common/metrics/gendoc/options.go

```diff
@@ -72,7 +72,8 @@ func FileOptions(f *ast.File) ([]interface{}, error) {
 				if !ok {
 					continue
 				}
-				if imports[ident.Name] != "github.com/hyperledger/fabric/common/metrics" {
```

Comment:
This is not only doc change, but some code change too.

Reply:
yes, without this change, there would be no documentation